packages/sync/src/config/SyncConfig.ts

```diff
@@ -36,6 +37,13 @@ export class SyncConfig implements DataSyncConfig {
    * Method used to join user configuration with defaults
    */
   public merge(clientOptions?: DataSyncConfig): DataSyncConfig {
+    if (clientOptions && clientOptions.conflictStrategy) {
```

Comment:
Not sure if the merge is the right place for doing this

Reply:
sure, I can do that. I think the logic is ok because if conflictStrategy is not passed in the clientOptions object it is merged with the one set here: https://github.com/aerogear/aerogear-js-sdk/blob/366ce6d636398b5e17e53423db9134c4bb5029fa/packages/sync/src/config/SyncConfig.ts#L24

Reply:
ok. So let's use this `conflictStrategy` inside here instead of defining a separate one. 
It will be a single place to make a change.